lib/widgets/svg_icon.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class SvgIcon extends StatelessWidget {
+  const SvgIcon(
+      this.iconPath
+      );
+
+  final String iconPath;
+
+  @override
+  Widget build(BuildContext context) {
+    return SvgPicture.asset(
+      iconPath,
+      color: IconTheme.of(context).color,
+      width: IconTheme.of(context).size,
+      height: IconTheme.of(context).size,
```

Comment:
Instead of 3 times call IconTheme.of(context) it would be nice to create local variable and retrieve properties from local object.
var iconThemeData = IconTheme.of(context);
And then in constructor - color =  iconThemeData.color